enhancements/aiops/20230120-Identify-prometheus-metrics-that-match-known-anomaly-chart-patterns.md

```diff
@@ -0,0 +1,57 @@
+# Title: 
+Identify prometheus metrics that match known anomaly chart patterns
+
+## Summary: 
+This enhancement proposal offers a new feature to match prometheus metrics to a set of known pre-defined anomaly chart patterns. This feature will make operators' life easier in the process of outage diagnosis.
+
+## Use case: 
+* Monitoring metrics in a multi-cluster setup is challenging. Operators could be responsible for hundreds of clusters and each cluster has tons of different metrics. When an outage occurs, it's difficult to identify root-cause metrics to assist outage diagnosis. Users need tools to help them reduce the scope and identify anomaly metrics faster.
+
+## Benefits: 
+* Makes operators' life easier in the process of outage diagnosis.
+* Improve user experience. Metric data will get condensed and categorized into insights that are much easier to digest.
+* Reduce mean time to resolve outages. This method will shorten the time operators taken to find out root cause.
+* Reduce economic loss.
+
+## Impact: 
+* No direct impact to existing system in terms of performance. However, initially this feature will only be deployed in `reactive mode` -- it's only triggered when an alert is fired. If this new feature is deployed in a `proactive mode`(which means it runs periodically) in the future, then it might require more resources as it would have to continuously compute for all metrics.
```

Comment:
Is it necessary to do this when an alert is fired? Can we initially just have this run with the click of a button?
Let's say a user wants to use this feature, they'd only be able to use it if an alert is setup. That is adding unnecessary overhead to even get started with this new feature.

Reply:
Yes we can start with the click of a button for user to get started with this new feature. It's just more ideal to trigger it with an alert.